documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+2. Location(s)
+   1. A location consists of the country, city, and venue.
+   2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
+   3. Competitions will be accepted if there are no other competitions within 100 km driving distance and less than 19 days away.
+      1. Otherwise, the WCA Board should review the circumstances of the proposed competition. A Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+         1. The two competitions don’t have events in common.
+         2. The two competitions take place in different countries.
+   4. 3x3x3 Fewest Moves simultaneous competitions
+      1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
+         1. If any location of such competitions is closer than 100 km to any other competition, then
+            1. both competitions should be at least 5 days apart, or
+            2. the other competition must not have the 3x3x3 Fewest Moves event.
+      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap.
+   5. Twin competitions
+      1. Twin competitions are WCA Competitions that are held less than 5 days apart for which the distance between them is less than 100 km, with the intention to accommodate more competitors to compete in one area.
+         1. Twin competitions may be accepted if the need for competitions in an area is so great that the community will benefit from having multiple competitions held in a short period of time.
+      2. Twin competitions are obliged to clearly and publicly state their condition of twin competitions and their special participation terms.
+         1. A competitor is only permitted to participate in one of the twin competitions.
+            1. If a competitor competes at more than one of the involved twin competitions, only the results from the competition that they compete in first will be kept.
+3. WCA Delegate(s)
+4. Organizer(s)
+   1. Organizers can be specific persons and/or organizations.
+5. Competitor limit
+   1. The competitor limit can be on either a "first come first serve" basis or based upon qualification results or rankings in the WCA world rankings of a previously announced calendar date.
+      1. Qualification for an event must be based only on results from the same event.
+   2. If the competitor limit for a competition or an event is reached before the competition, the organization team must not approve any additional registrations for the competition or the event.
+   3. The competitor limit is per competition. Per day competitor limits are not allowed.
```

Comment:
```suggestion
   3. The competitor limit is per competition. Per day competitor limits are not allowed.
   4. The WCA Competition Announcement Team will review the competitor limit and/or qualification results or rankings to ensure that they are utilized properly.
```

Without something like this WCAT looses the ability to deny a competitor limit of 10 in NY, or a qualification time of 6 seconds in 3x3.

Reply:
Do you think Motion 10.2019.1 is too broad?
2. The WCA Competition Announcement Team has the following rights and duties:
2.1 Announcing WCA Competitions:
2.1.1 Approving and announcing WCA Competitions.
2.1.1.1 Ensuring announcements adhere to WCA quality standards.
2.1.2 Rejecting competitions.

Reply:
If a change is made about this, I'd prefer it be in 10.2019.1 instead of this policy, but I think 2.1.1.1 reasonably allows rejection of the examples you described.

Reply:
Hmmm, reading the document as a whole without the comments, I think that the document makes it sound like these items (and the optional requirements) will be accepted without any pushback from WCAT, whenever we would not accept things, most of the time related in these areas, due to them not being acceptable.

Reply:
I agree with Jacob here. From a delegate's and organisers perspective, it is better to have all of the requirements in one place, rather than having to cross reference the motions and the policy.

Reply:
That sounds reasonable to me too.